packages/jest-worker/src/workers/messageParent.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {PARENT_MESSAGE_CUSTOM} from '../types';
+
+const isWorkerThread = () => {
+  try {
+    const {isMainThread, parentPort} = require('worker_threads');
+    return !isMainThread && parentPort;
+  } catch (_) {
```

Comment:
```suggestion
  } catch {
```

but what's this guard for? The fact the module is missing in node 8? If so, would this code path ever be called - wouldn't earlier guards catch it?